app.html

```diff
@@ -1,22 +1,41 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-
-<head>
-    <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
     <title>Contacts List</title>
-    <link rel="stylesheet" href="app.css">
-</head>
+    <link rel="stylesheet" href="app.css" />
+  </head>
 
-<body>
+  <body>
     <div id="app">
+      <div id="contact-list-container" class="flex-child">
```

Comment:
remove the class for now, its not needed and we should never use those names like "flex-child", since we're trying to pretend this is for production